src/components/twilio-voice-monitoring/routes.js

```diff
@@ -40,16 +44,20 @@ router.post('/twiml', Twilio.webhook({ protocol: 'https' }, authToken), (req, re
 // Validate incoming Twilio requests
 // https://www.twilio.com/docs/usage/tutorials/how-to-secure-your-express-app-by-validating-incoming-twilio-requests
 router.post('/conference-events', Twilio.webhook({ protocol: 'https' }, authToken), async (req, res) => {
```

Comment:
This callback doesn't need to be async anymore since you wrapped the inner asynchronous logic into a lambda.